src/logger.rs

```diff
@@ -110,11 +113,21 @@ impl Logger {
             StdoutLog::Off => {}
         }
 
-        if let Some(log_gfx_device_level) = config.log_gfx_device_level {
-            logger.dispatch = logger
-                .dispatch
-                .level_for("gfx_device_gl", log_gfx_device_level);
-        }
+        let log_gfx_backend_level = config.log_gfx_backend_level.unwrap();
```

Comment:
Hiya, could you use `if let Some(log_gfx_backend_level) = config.log_gfx_backend_level { .. }`, and similar for `log_gfx_rendy_level`?
It's a valid scenario to construct `LoggerConfig` with them as `None`.

Thanks :bowing_man:!

Reply:
Sure thing, I'll do that!